include/link/main.h

```diff
@@ -27,16 +27,18 @@ extern bool beVerbose;
 extern bool isWRA0Mode;
 extern bool disablePadding;
 
+enum FileStackNodeType {
```

Comment:
Shouldn't this go to `linkdefs.h`?

Reply:
Yes, two files had the same anonymous enum; good catch.